### DISCUSSION TITLE
aggiornamento dash

### DISCUSSION THREAD

**Discussion by Robertomastro** [indomus/forum#1128] [HUB: Home Assistant]
Ciao a tutti,
mi sono perso alcune configurazioni sulla dashboard di home assistant
![image](https://user-images.githubusercontent.com/45659710/144072025-fa4bac12-78f6-4595-ae7e-8d0346781de9.png)
come posso riabilitare le entità non disponibili?
Non funzionano neppure le macro su node-red.
grazie mille

**Comment by snakuzzo** [comment]
Se non sono disponibili, vuol dire che non possono essere utilizzate u lovelace.
Quindi devi ricontrollare la configurazione...controlla i log e come hai aggiunto/integrato quelle enittà. Noi non possiamo esserti d'aiuto
